src/components/common/NavBar.jsx

```diff
@@ -1,66 +1,95 @@
 import React from "react";
-import { Nav, Navbar } from "react-bootstrap";
+import { Nav, Navbar, Button, Image } from "react-bootstrap";
 import "./index.css";
 import { Link } from "react-router-dom";
 import LangDropdown from "./LangDropdown";
 
 export const NavBar = () => {
+  const [navItemsPosition, setNavItemsPosition] = React.useState("-100%");
```

Comment:
usually we don't use css in js like this. instead this should be

``[isHamburgerShown, setIsHamburgerShown]```

and it is a boolean

then you add a classname to whatever you need depending on this